.github/workflows/shippable_builds.yml

```diff
@@ -224,21 +224,45 @@ jobs:
 
           ./gradlew :app-${APP_NAME}:printVersionInfo -PbuildType=${RELEASE_TYPE} -PflavorName=${PACKAGE_FLAVOR} --configure-on-demand
 
-      - name: Bump version code
-        id: bump_version_code
-        if: ${{ contains(matrix.releaseTarget, 'github') }}
+      - name: Determine new version code
+        id: new_version_code
+        if: ${{ contains(matrix.releaseTarget, 'github') || needs.dump_config.outputs.releaseType == 'daily' }}
         shell: bash
         env:
           APP_NAME: ${{ matrix.appName }}
           OLD_VERSION_CODE: ${{ steps.appinfo.outputs.VERSION_CODE }}
+          RELEASE_TYPE: ${{ vars.RELEASE_TYPE }}
         run: |
-          NEW_VERSION_CODE=$(($OLD_VERSION_CODE + 1))
+          if [[ "$RELEASE_TYPE" = "beta" || "$RELEASE_TYPE" = "release" ]]; then
+            NEW_VERSION_CODE=$(($OLD_VERSION_CODE + 1))
+          elif [[ "$RELEASE_TYPE" = "daily" ]]; then
+            VERSION_FILE="$(pwd)/version.properties"
```

Comment:
Does this require another PR? I don't think we have this file yet

Reply:
I'm only using that file in order to have parallel matrix jobs use the same version code, which may not be necessary.